core/shared/platform/ego/platform_internal.h

```diff
@@ -0,0 +1,6 @@
+/*
```

Comment:
How about `core/shared/platform/open-enclave`?

Reply:
The specification is from EGO not Open Enclave.